backend/tests/integration/records/serializers/test_bibtex.py

```diff
@@ -193,3 +196,13 @@ def test_bibtex_encodes_unicode_outside_of_math_environments(inspire_app):
     response_data = response.get_data(as_text=True)
     assert expected_status_code == response_status_code
     assert expected_result == response_data
+
+
+def test_literature_detail_bibtex_link_alias_format(inspire_app):
+    expected_status_code = 200
+    record = create_record("lit")
+    expected_content_type = "application/x-bibtex"
+    with inspire_app.test_client() as client:
+        response = client.get(f"/literature/{record['control_number']}?format=bibtex")
+    assert response.status_code == expected_status_code
+    assert response.content_type == expected_content_type
```

Comment:
can you maybe test that it actually returns a bibtex payload?

Reply:
Is it necessary? We have only one way to return bibtex content_type. And we are testing it's output in other tests. Don't forget that this `format=` stuff is only a shortcut.